lib/commands/new/hosted-skill-helper.js

```diff
@@ -92,7 +92,6 @@ function createHostedSkill(hostedSkillController, userInput, vendorId, callback)
             }
             skillMetadataController.enableSkill((enableErr) => {
                 if (enableErr) {
-                    Messenger.getInstance().error(enableErr);
```

Comment:
This one is a little out of the context, but I noticed when I run unit test. This error will anyway be displayed in the upstream, so it's duplicate